base/applications/network/ftp/ftp.c

```diff
@@ -73,7 +73,11 @@ int	ptflag = 0;
 int	allbinary;
 struct	sockaddr_in myctladdr;
 uid_t	getuid();
+#ifdef __REACTOS__
+void lostpeer(void);
+#else
 sig_t	lostpeer();
+#endif
```

Comment:
This is a weird one.
Do we sync it with anything?

Reply:
Hmm, history suggests we never synced it. I just noticed it's 3rd party code so I added those ReactOS ifdefs.

Reply:
it's a weird construction either way, but I guess this wont do any harm.